smbackend/settings.py

```diff
@@ -217,6 +217,11 @@ def gettext(s):
 # Shortcut generation URL template
 SHORTCUTTER_UNIT_URL = env("SHORTCUTTER_UNIT_URL")
 
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.turku.fi"
+EMAIL_HOST_USER = "varaamo@turku.fi"
```

Comment:
Switch to env value and request different address for this project

Reply:
Fixed